test/e2e/operator_test.go

```diff
@@ -299,10 +297,10 @@ func TestClusterIngressUpdate(t *testing.T) {
 		t.Fatalf("failed to get CA certificate configmap: %v", err)
 	}
 
-	originalSecret := ci.Spec.DefaultCertificateSecret
+	originalSecret := ci.Spec.DefaultCertificate
```

Comment:
Using `DeepCopy()` here would feel more correct.  (`DeepCopy()` will return a nil value if the input is nil, so the logic would still be correct.)

Reply:
Fixed